pages/instructorAssessmentUploads/csvHelpInstanceQuestionScores.ejs

```diff
@@ -1,25 +1,19 @@
 <p>
-  Upload a CSV file like this:
+  Upload a CSV file in the format of
```

Comment:
Perhaps add a link out to the documentation for manual grading?

Reply:
done